package.json

```diff
@@ -33,26 +33,32 @@
     "scroll-into-view-if-needed": "^2.2.16"
   },
   "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-transform-flow-strip-types": "^7.0.0",
+    "@babel/plugin-transform-object-assign": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/template": "^7.0.0",
+    "@babel/types": "^7.0.0",
     "autoprefixer": "^9.1.3",
     "axios": "^0.18.0",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.6",
-    "babel-jest": "^23.4.2",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-flow-react-proptypes": "^24.1.1",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
I think this is an older version. Is it worth looking at 7.2.2?

Reply:
It looks like this is the latest version I can install
```
  '6.26.2',
  '6.26.3',
  '7.0.0-alpha.1',
  '7.0.0-alpha.2',
  '7.0.0-alpha.3',
  '7.0.0-alpha.6',
  '7.0.0-alpha.7',
  '7.0.0-alpha.8',
  '7.0.0-alpha.9',
  '7.0.0-alpha.10',
  '7.0.0-alpha.11',
  '7.0.0-alpha.12',
  '7.0.0-alpha.14',
  '7.0.0-alpha.15',
  '7.0.0-alpha.16',
  '7.0.0-alpha.17',
  '7.0.0-alpha.18',
  '7.0.0-alpha.19',
  '7.0.0-alpha.20',
  '7.0.0-beta.0',
  '7.0.0-beta.1',
  '7.0.0-beta.2',
  '7.0.0-beta.3',
  '7.0.0-bridge.0' ]
```

Reply:
Looks like they moved to `@babel/core`: https://www.npmjs.com/package/@babel/core

Reply:
Yea lemme see if i can remove the old one!